<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ИСПОЛЬЗОВАНИЕ МЕТОДОВ ПАРОЛЬНОЙ ЗАЩИТЫ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Выполнил:
Студент группы АБс-123,
Борискиина Алёна Валерьевна</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
    background-color: #ffffff;
    border-radius: 10px;
};</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources/mainwindow.qrc">:/resources/images/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>361</width>
       <height>176</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_login">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #ccc;
    border-radius: 10px;
    padding: 8px;
    background-color: white;
}</string>
        </property>
        <property name="placeholderText">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid #ccc;
    border-radius: 10px;
    padding: 8px;
    background-color: white;
}</string>
        </property>
        <property name="placeholderText">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_enter">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    border: 2px solid #ccc;
    border-radius: 10px;
    padding: 8px;
	color:white;
    background-color: #00cc73;
}</string>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_register">
        <property name="styleSheet">
         <string notr="true">QPushButton{
    border: 2px solid #ccc;
    border-radius: 10px;
    padding: 8px;
	color:white;
    background-color: #90ab50;
}</string>
        </property>
        <property name="text">
         <string>Зарегистрироваться</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_quit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>300</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: 2px solid #ccc;
    border-radius: 10px;
    padding: 8px;
	color:white;
    background-color: #861f3d;
}</string>
     </property>
     <property name="text">
      <string>Закончить работу</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources/mainwindow.qrc"/>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
